content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md

```diff
@@ -90,7 +90,7 @@ Signs that your API might not be declarative include:
 
 ## Should I use a configMap or a custom resource?
 
-Use a ConfigMap if any of the following apply:
```

Comment:
The original text is more idiomatic English.

Reply:
Oh... I'll revert that then.